lib/tasks/post_migration_index_checker.rake

```diff
@@ -3,14 +3,15 @@
 Rake::Task["db:migrate"].enhance do
   tables = ActiveRecord::Base.connection.tables
   all_foreign_keys =
-    tables
-      .flat_map do |table_name|
-        ActiveRecord::Base
-          .connection
-          .columns(table_name)
-          .map { |c| [table_name, c.name].join(".") }
-      end
-      .select { |c| c.ends_with?("_id") }
+    yolo = tables
```

Comment:
I needed to assign this inline statements return value to a variable to make `rubocop` happy. And I think robucop has a point. Kinda hard to understand that appended `.METHOD` call. I have no idea what the code does or what a better name for the variable would be. How would you call it, @phoet? My background is in JS/TS.

Reply:
Please advise on the variable name here @phoet

Reply:
what exactly is rubocop complaining about? there is no need for any intermediary assignment. this is basically a one-liner that was auto-formatted in some weird way.

Reply:
I tried to fix it. Please check the code. Maybe just check out the branch and fix this part quickly yourself? I think you can push to my branch.